router.go

```diff
@@ -1003,6 +1033,9 @@ func Router(addr string) *echo.Echo {
 		g.GET("/:app/:version/screenshots/*", getVersionScreenshot)
 		g.HEAD("/:app/:version/tarball/:tarball", getVersionTarball)
 		g.GET("/:app/:version/tarball/:tarball", getVersionTarball)
+
+		ug.GET("/.well-known/:filename", universalLink, middleware.Gzip())
+		ug.GET("/:slug", universalLinkRedirect)
```

Comment:
I don't think we should have those routes for each space, but only once at root (like for editors). In particular, I think that `.well-known` must be at root.